config/routes.rb

```diff
@@ -6,19 +6,8 @@
     # Static pages
     get "/static_pages/home", to: "static_pages#home", as: "home"
 
-    # Đăng ký (Sign up)
-    get "/signup", to: "users#new"
-    post "/signup", to: "users#create"
-    put "/users/:id", to: "users#edit"
-
-    # Đăng nhập/Đăng xuất (Login/Logout)
-    get "/login", to: "sessions#new"
-    post "/login", to: "sessions#create"
-    delete "/logout", to: "sessions#destroy"
-
-    # Kích hoạt tài khoản (Account Activations)
-    resources :account_activations, only: :edit
-
+    
+    devise_for :users
```

Comment:
```suggestion
    devise_for :users, only: [:sessions]
```
Giới hạn các router lại